docs: document filterable attributes (#1996)
mendersoftware/mender-docs: 02.Overview/08.Inventory/docs.md

For more information on customizing the Inventory attributes,
please refer to the [client configuration section](../../03.Client-installation/04.Inventory/docs.md).

## Limitations

> Each device can upload up to 100 inventory attibutes in a single inventory update.

Author: fixed

> Only first 100 of different inventory attributes uploaded by the devices are indexed by the Mender backend
> and can be used to search for devices.

Reviewer: ```suggestion
When indexing, the Mender server considers up to 100 different inventory attributes' names uploaded by the devices. It will ignore all the additional inventory attributes' names. 
```

Author: fixed

> You can see current usage and list of filterable attributes in Global settings.

Reviewer: ```suggestion
You can see the current usage and the list of filterable attributes in the Global settings.
```

Author: fixed

![Global settings](global-settings.png)

> If you reached the limit of filterable attributes as you can see below,

Reviewer: ```suggestion
If you reached the limit of filterable attributes, as you can see below:
```

Author: fixed

![limit reached](limit-reached.png)

> but your use case requires different set of attributes please email [contact@mender.io](mailto:contact@mender.io).

Reviewer: ```suggestion
but your use case requires a different set of attributes, please email [contact@mender.io](mailto:contact@mender.io).
```

Author: fixed
